server/src/models/Users/User.model.js

```diff
@@ -9,31 +10,31 @@ const UserSchema = new Schema({
     type: String,
     required: true,
     index: { unique: true },
-    lowercase: true,
+    lowercase: true
   },
   password: {
     type: String, required: true
   }
 })
 
-UserSchema.pre('save', function(next) {
+UserSchema.pre('save', function (next) {
   const user = this
 
   if (!user.isModified('password')) return next()
 
-  bcrypt.genSalt(config.authentication.salt_factor, function(err, salt) {
-      if (err) return next(err)
+  bcrypt.genSalt(config.authentication.salt_factor, (err, salt) => {
+    if (err) return next(err)
 
```

Comment:
Testoval si toto? ja mam tuseni ze jsem se tu pokousel dat arrow function a ono to nefachalo